packages/frontend/src/components/campaigns/campaign/styles.module.css

```diff
@@ -0,0 +1,9 @@
+.root {
+    composes: row from "../styles.module.css";
+    @apply bg-white 
+        hover:bg-gray-200
+        cursor-pointer
+        transition-colors
+        duration-200
+        rounded-xl h-14;
```

Comment:
I'd add the `ease-in-out` class here.

Reply:
Addressed in e3a2263d1d5914abd841a5695dcc92f0d2856e17